client/src/modules/stock/aggregated_consumption/aggregated_consumption.js

```diff
@@ -228,6 +233,21 @@ function StockAggregatedConsumptionController(
       });
   }
 
+  function loadCurrentInventories(depot, dateTo = new Date()) {
+    vm.loading = true;
+    Stock.lots.read(null, { depot_uuid : depot.uuid, dateTo })
+      .then(lots => {
+        vm.currentInventories = lots.filter(item => item.quantity > 0);
+
+        // Here we check directly if a Depot has inventories in stock available in current date
+        vm.emptyCurrentStock = !vm.currentInventories.length;
```

Comment:
I think something like this would more readable:

vm.emptyCurrentStock = vm.currentInventories.length <= 0;

Reply:
I remove **vm.emptyCurrentStock** this element, I did not use it anywhere

Reply:
Nice!